runtime/lua/vim/lsp/completion.lua

```diff
@@ -507,7 +507,13 @@ local function trigger(bufnr, clients, ctx)
     local server_start_boundary --- @type integer?
     for client_id, response in pairs(responses) do
       if response.err then
-        vim.notify_once(response.err.message, vim.log.levels.WARN)
+        local client = lsp.get_client_by_id(client_id)
+        local msg = ('%s: %s %s'):format(
+          client and client.name or '',
+          response.err.code or '',
```

Comment:
I think that something other than empty strings would be more useful placeholders:
```suggestion
          client and client.name or 'UNKNOWN',
          response.err.code or 'NO_CODE',
```